src/styles/themeManager.js

```diff
@@ -0,0 +1,257 @@
+import merge from 'lodash.merge';
+import update from 'react-addons-update';
+import Colors from './colors';
+import ColorManipulator from '../utils/color-manipulator';
+import extend from '../utils/extend';
+import lightBaseTheme from './baseThemes/lightBaseTheme';
+import zIndex from './zIndex';
+
+/**
+ * get the MUI theme corresponding to a base theme.
+ */
+export function getMuiTheme(baseTheme) {
```

Comment:
I like your idea :+1: :+1: yeah, I'll figure out a clean way to do this. thanks :+1: